config/dossier.yml

```diff
@@ -0,0 +1,21 @@
+development:
+  adapter: mysql2
+  database: onebody_dev
+  host: localhost
+  username: onebody
+  password: onebody
+  encoding: utf8
+test:
+  adapter: mysql2
+  database: onebody_test
+  host: localhost
+  username: onebody
+  password: onebody
+  encoding: utf8
+production:
+  adapter: mysql2
+  database: onebody
+  host: localhost
+  username: onebody
+  password: onebody
+  encoding: utf8
```

Comment:
Any reason you changed this? If this is needed, we'll need to update install scripts and install documentation so people know to update it.